enterprise_gateway/services/processproxies/k8s.py

```diff
@@ -239,7 +248,7 @@ def _determine_kernel_namespace(self, **kwargs: dict[str, Any] | None) -> str:
 
         # If KERNEL_NAMESPACE was provided, then we assume it already exists.  If not provided, then we'll
         # create the namespace and record that we'll want to delete it as well.
-        namespace = kwargs["env"].get("KERNEL_NAMESPACE")
+        namespace = os.environ.get("KERNEL_NAMESPACE")
```

Comment:
Prior to launch, `KERNEL_NAMESPACE` must be pulled from the start request arguments (`kwargs["env"]`) rather than the EG process env, so this change needs to be reverted.
```suggestion
        namespace = kwargs["env"].get("KERNEL_NAMESPACE")
```